src/cascade/dismod/db/metadata.py

```diff
@@ -335,6 +335,14 @@ class Option(Base):
     option_value = Column(String(), nullable=False)
 
 
+class Metadata(Base):
+    __tablename__ = "c_metadata"
+
+    c_metadata_id = Column(Integer(), primary_key=True, autoincrement=False)
+    key = Column(String(), unique=True)
+    value = Column(String(), nullable=False)
+
```

Comment:
This works. Metadata seems like it could refer to something dismod_at creates, or something about the model. Maybe c_execution_data, or something.

Reply:
Yeah, that makes sense.